index.js

```diff
@@ -0,0 +1,21 @@
+
+const register = function (server, options, next) {
+  console.log('registering tags');
```

Comment:
console.log is helpful for debuging but you should delete them on production code
see the point 3 of https://codequs.com/p/Syo7zYOtle/10-things-you-should-do-while-running-node-js/